packages/fxa-settings/src/models/FlowContext.ts

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'react';
+
+export const FlowContext = createContext<any>(undefined);
```

Comment:
This could use a license at the top.

Also, seems like we could use better typing than just `any` here. Are we doing that in a follow up?

Also curious about creating a new provider vs just adding this to the existing `AppContext`, or not putting it in Context at all, see [React docs](https://reactjs.org/docs/context.html#before-you-use-context). Are we going to be prop drilling this value without it?

Reply:
So I had to put it into a Context bc now that the Settings app is router component receiving a path, it can't take in props. I'm very open to moving it into a different context tho! (Or another solution that I haven't thought of!)

Reply:
Per Slack, we're going to file a follow up here to investigate and either type this better or remove it.